packages/lesswrong/components/localGroups/LocalGroupPage.tsx

```diff
@@ -344,16 +348,16 @@ const LocalGroupPage = ({ classes, documentId: groupId }: {
   }
   
   let pastEventsList: JSX.Element|null = <>
-    <Components.Typography variant="headline" className={classes.eventsHeadline}>
+    <Typography variant="headline" className={classes.eventsHeadline}>
       Past Events
```

Comment:
```suggestion
      Past events
```

Reply:
This is the non-EAF version, so keeping it title cased